code/game/atoms_movable.dm

```diff
@@ -1193,3 +1193,12 @@
 	target = get_turf(target_loc)
 
 	current_tick = 1
+
+
+/atom/movable/proc/speen()
```

Comment:
Fuck you

Reply:
Look, there are things already implementing `spin()`, I was strong-armed into this